RELEASING.md

```diff
@@ -0,0 +1,7 @@
+The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [maintainers](MAINTAINERS.md).
+
+1. Create a tag, e.g. v2.1.0, and push it to the GitHub repo.
+1. The [release-drafter.yml](.github/workflows/release-drafter.yml) will be automatically kicked off and a draft release will be created.
+1. This draft release triggers the [jenkins release workflow](ADD CORRESPONDING LINK HERE) as a result of which opensearch-benchmark client is released on [PyPi](https://pypi.org/project/opensearch-benchmark/).
```

Comment:
You can below link here:
https://build.ci.opensearch.org/job/opensearch-benchmark-release